### PR TITLE
:feat integrating account balance

### PR DESCRIPTION
**Pull request by svenvoskamp**
# Changes

- Adding `wui-list-token` composite
- Adding `wui-list-description` composite
- Added switch tab logic to show right content for chosen tab 
- Adjusted `wui-tabs` composite to fix state in account view
- Adjusted `wui-tag` composite
- Added `w3m-account-activity-widget` for activity tab content
- Added `w3m-account-nft-widget` for nft tab content
- Added `w3m-account-token-widget` for nft token content
- Added `w3m-activity-list` to share ui/logic for activity content 
- Added `getBalance` function to `BlockChainApiController`
- Added `fetchTokenBalance` function to `AccountController`
- Added `tokenBalance` state to `AccountController`

### CODE REVIEW

File: packages/scaffold/src/partials/w3m-account-wallet-features-widget/index.ts [hunk 1]
```diff
@@ -93,11 +100,41 @@ export class W3mAccountWalletFeaturesWidget extends LitElement {
         <wui-tooltip-select text="Send" icon="send"></wui-tooltip-select>
       </wui-flex>
 
-      <wui-tabs localTabWidth="120px" .tabs=${ConstantsUtil.ACCOUNT_TABS}></wui-tabs>
+      <wui-tabs
+        .onTabChange=${this.onTabChange.bind(this)}
+        .activeTab=${this.currentTab}
+        localTabWidth="104px"
```
Comment (tomiir) [inline packages/scaffold/src/partials/w3m-account-wallet-features-widget/index.ts:106]: is this ok ?

**Review by tomiir — APPROVED** [review]
Looks good, though I don't really understand how `fetchGetBalance` works

File: packages/scaffold/src/partials/w3m-account-wallet-features-widget/index.ts (hunk 1 above)
Comment (svenvoskamp) [reply packages/scaffold/src/partials/w3m-account-wallet-features-widget/index.ts:106]: Yes normally `localTabWidth` is `100px`, but we set `wui-tabs` to `width:100%`, so `100px` is not enough anymore.

**Comment by svenvoskamp** [comment]
> Looks good, though I don't really understand how `fetchGetBalance` works

It will make a call to the `get` method from the `FetchUtil`. It will create the url based on the arguments passed in. Then it will fetch this url and return the response.

**Pull request merged**